sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -400,6 +439,12 @@ public class UID2Manager internal constructor(
         private const val UID2_API_URL_KEY = "uid2_api_url"
         private const val UID2_API_URL_DEFAULT = "https://prod.uidapi.com"
 
+        // The metadata keys that are used to provide access to required parameters for client side token generation.
```

Comment:
_The metadata keys that are used to provide access to required parameters for Client Side Integration_ 
?